tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot

```diff
@@ -154,8 +154,11 @@ Container on Open Network And Port Forwarding Persist After Reboot
     [Setup]     NONE
 
     Log To Console  Create Port Groups For Container network
+    ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.portgroup.remove open-net
+    Run Keyword If  '%{HOST_TYPE}' == 'VC'  Remove VC Distributed Portgroup  open-net
+
     ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.portgroup.add -vswitch vSwitchLAN open-net
-    ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Add VC Distributed Portgroup  test-ds  open-net
+    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run And Return Rc And Output  govc dvs.portgroup.add -vlan=196 -dvs test-ds open-net
```

Comment:
This will not work if run outside of the HaaS environment. All of this infrastructure config should be pulled out into a separate setup mechanism, robot or otherwise, rather than inline in the tests.

The tests should at most have a pre-condition check so they fail if the necessary infrastructure elements are not available.

Given this is blocking CI right now I'm not going to block the merge conditional on a follow up issue to refactor the infrastructure setup out of the individual tests.

Reply:
https://github.com/vmware/vic/issues/7462